config/application.rb

```diff
@@ -124,9 +124,8 @@ class Application < Rails::Application
       #   dev/test/prod in the config revamp.
       ActiveRecord::Base.logger.level = Logger::DEBUG
 
-      Rails::ConsoleMethods.class_eval do
-        include Vmdb::ConsoleMethods
-      end
+      Rails::ConsoleMethods.include(Vmdb::ConsoleMethods)
+      TOPLEVEL_BINDING.eval('self').extend(Vmdb::ConsoleMethods) if defined?(Pry)
```

Comment:
Wow, good find but that's awful.  Does pry provide a less awful hook to add methods into your pry session?